app/src/main/kotlin/PushYourLuck/PushYourLuckAction.kt

```diff
@@ -0,0 +1,12 @@
+package PushYourLuck
+
+enum class PushYourLuckAction {
+    ROLL, CASHOUT;
+
+    override fun toString(): String {
```

Comment:
Why is this overriden? By default, the name of the enum will be what toString returns and the only difference is uppercase vs lowercase. Unless you really like lowercase, I don't see this override as necessary.